kolibri_explore_plugin/collectionviews.py

```diff
@@ -910,7 +927,7 @@ def resume_download(request):
     Returns download status.
     """
 
-    saved_state = request.session.get("COLLECTIONS_STATE")
+    saved_state = _get_saved_collections_state(request)
```

Comment:
It's a preëxisting bug, but: `_get_saved_collections_state` validates that the state contains at least the following keys:

```
    collections_state_keys = (
        "collection_name",
        "collection_sequence",
        "stage",
    )
```

but the block below, inside the `try`, uses `saved_state["name"]` and `saved_state["sequence"]`. One of these two is surely wrong! I think the block below was missed in 88360791.

Reply:
In sorting that out, I noticed a _different_ problem, resolved in d4db24e5 :)